### PR TITLE
add files

### PR DESCRIPTION
**Pull request by VasylRosokha**
### Hooli-style Popup

[Demo](https://vasylrosokha.github.io/Hooli-style-Popup/) |
[Code base](https://github.com/VasylRosokha/Hooli-style-Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by VasylRosokha** [comment]
I  have completed self-checks

File: submissions/VasylRosokha/Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Popup</title>
+</head>
+<body>
+
+    <header class="header">
```
Comment (A-Ostrovnyy) [inline submissions/VasylRosokha/Popup/index.html:12]: Add a mobile version of your popup.
image

File: submissions/VasylRosokha/Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/photos.png" alt="" class="menu-item-img">Photo</a>
+                                </li>
+                                <li class="menu-item">
+                                    <a href="#" class="menu-item-link"><img src="img/icons/documents.png" alt="" class="menu-item-img">Document</a>
+                                </li>
+                            </ul>
+    
+                            <input type="checkbox" id="popup-more" class="more">
+                            <label for="popup-more" class="more-label">More</label>
```
Comment (A-Ostrovnyy) [inline submissions/VasylRosokha/Popup/index.html:63]: Logically when the second part of the popup is open this button should be hidden.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/VasylRosokha/Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    
+                            <div class="more-menu">
```
Comment (A-Ostrovnyy) [inline submissions/VasylRosokha/Popup/index.html:65]: Avoid horizontal scroll except if you need it
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue.

**Comment by VasylRosokha** [comment]
> One last issue.

What I need to do?

**Comment by A-Ostrovnyy** [comment]
> > One last issue.
> 
> What I need to do?

Remove horizontal scroll when the second part is open

File: submissions/VasylRosokha/Popup/css/style.css [hunk 4]
```diff
@@ -136,7 +141,7 @@ li {
     .menu, .more-menu {
         width: 300px;
     }
-    .menu-list {
+    .menu-list, .more-list {
         gap: 5px;
     }
 }
```
Comment (A-Ostrovnyy) [inline submissions/VasylRosokha/Popup/css/style.css:147]: Fo the future put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**